priv/repo/migrations/20181113033726_add_account_to_room_objects.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Ret.Repo.Migrations.AddAccountToRoomObjects do
+  use Ecto.Migration
+
+  def change do
+    alter table("room_objects") do
+      add(:account_id, references(:accounts, column: :account_id))
```

Comment:
I suggest we make this `NOT NULL` and either:

- Nuke all the existing pinned objects (debatable, probably at this point a bad idea)
- Set all the legacy ones to account id of 0

The `NOT NULL` will help reduce the risk of bugs causing privacy holes since if we fail to capture that account id properly there is no way to purge the data if you lose the URL